docs/guides/references/_changelogs/12.4.0.md

```diff
@@ -0,0 +1,49 @@
+## 12.4.0
+
+_Released 1/24/2023_
+
+**Features:**
```

Comment:
Could consider doing a little bit of sorting here, pull the most impactful features to the top. Nothing super scientific, but the two feature flags (especially the memory related one!) and `.as()` options are probably more important than linebreaks in log messages.

Reply:
reordered in 4ba7d032b